index.html

```diff
@@ -0,0 +1,90 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;700&display=swap" rel="stylesheet">
+    <script src="https://kit.fontawesome.com/162dfb3961.js" crossorigin="anonymous"></script>
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+    <title>Youtube Page</title>
+  </head>
+<body>
+  <div class="wrapper">
+      <header id="header">
+          <div class="logo left">
+              <i id="menu" class="material-icons header-icons">menu</i>
+              <a href="#">
+                  <img src="images/youtube logo.png" alt="youtube logo">
+              </a>
+          </div>
+          <div class="search center">
+              <form action="">
+                  <input type="text" placeholder="Search">
+                  <button><i class="material-icons header-icons">search</i></button>
+              </form>
+              <i class="material-icons mic header-icons">mic</i>
+          </div>
+          <div class="icons right">
+              <i class="material-icons header-icons">videocam</i>
+              <i class="material-icons header-icons">apps</i>
+              <i class="material-icons header-icons">notifications</i>
+              <i class="material-icons display-this header-icons">account_circle</i>
+          </div>
+      </header>
+      <div class="main-wrapper">
+          <div class="left"></div>
```

Comment:
Please use the section element. it will make your code understandable.

Reply:
- [ ] I think it would be great if you mention the reason in clear way, so why you are asking for using section tag instead of div you can see it would be great to make your Html page semantic ✅🔥🔥